.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -22,6 +22,8 @@ body:
           required: false
         - label: This feature is not going to conflict with many of the existing options.
           required: false
+        - label: I have checked other issues (opened or closed) so I know that it's no repeated
+          required: false
```

Comment:
Shouldn't it be like this? required: true

Reply:
It may be. If it's required you can't send the issue, so maybe it's a good option